compiler/circle2circle/src/Circle2Circle.cpp

```diff
@@ -262,164 +262,97 @@ int entry(int argc, char **argv)
     return 255;
   }
 
+  // clang-format off
+  std::map<std::string /* option string */, Algorithms /* option enum */> option_str_to_enum;
+  option_str_to_enum["fold_add_v2"] = Algorithms::FoldAddV2;
+  option_str_to_enum["fold_cast"] = Algorithms::FoldCast;
+  option_str_to_enum["fold_densify"] = Algorithms::FoldDensify;
+  option_str_to_enum["fold_dequantize"] = Algorithms::FoldDequantize;
+  option_str_to_enum["fold_dwconv"] = Algorithms::FoldDepthwiseConv2D;
+  option_str_to_enum["fold_fully_connected"] = Algorithms::FoldFullyConnected;
+  option_str_to_enum["fold_gather"] = Algorithms::FoldGather;
+  option_str_to_enum["fold_mul"] = Algorithms::FoldMul;
+  option_str_to_enum["fold_reshape"] = Algorithms::FoldReshape;
+  option_str_to_enum["fold_shape"] = Algorithms::FoldShape;
+  option_str_to_enum["fold_sparse_to_dense"] = Algorithms::FoldSparseToDense;
+  option_str_to_enum["fold_squeeze"] = Algorithms::FoldSqueeze;
+  option_str_to_enum["forward_reshape_to_unaryop"] = Algorithms::ForwardReshapeToUnaryOp;
+  option_str_to_enum["forward_transpose_op"] = Algorithms::ForwardTransposeOp;
+  option_str_to_enum["fuse_activation_function"] = Algorithms::FuseActivationFunction;
+  option_str_to_enum["fuse_horizontal_fc_layers"] = Algorithms::FuseHorizontalFullyConnected;
+  option_str_to_enum["fuse_batchnorm_with_conv"] = Algorithms::FuseBatchNormWithConv;
+  option_str_to_enum["fuse_add_to_fullyconnected_bias"] = Algorithms::FuseAddToFullyConnectedBias;
+  option_str_to_enum["fuse_add_with_conv"] = Algorithms::FuseAddWithConv;
+  option_str_to_enum["fuse_add_with_fully_connected"] = Algorithms::FuseAddToFullyConnectedBias;
```

Comment:
Q: shouldn't it be like below?

I met a correctness issue from an internal quantized model after I updated `one-compiler`.
(FYI, e.g., `one-compiler=1.29.0~24082216` was fine.)

```diff
-  option_str_to_enum["fuse_add_with_fully_connected"] = Algorithms::FuseAddToFullyConnectedBias;
+  option_str_to_enum["fuse_add_with_fully_connected"] = Algorithms:: FuseAddWithFullyConnected;
```

https://github.com/Samsung/ONE/blob/daf9a85b154f7d67c7047d156a1018651d0d3000/compiler/one-cmds/how-to-use-one-commands.txt#L167
https://github.com/Samsung/ONE/blob/daf9a85b154f7d67c7047d156a1018651d0d3000/compiler/one-cmds/how-to-use-one-commands.txt#L169

Reply:
@ejjeong , plz fire fix PR :)